### ISSUE TITLE
CKCC Recompute through Core RNASeq & QC 3.2.1

### ISSUE THREAD

**Issue by klearned** [UCSC-Treehouse/operations#41]
This is Data team 1st priority!

We're going to run all the CKCC Partner Sample data through the Core RNASeq/QC Pipeline 3.2.1:
- Stanford (human samples)
- PNOC 
- CHOC (pre-NANT)
- CHOC/NANT
- Alejandro's latest if we get it in time (Katrina will call Alex)

Plan:

1. Create a single manifest for all the samples
2. Create ckccPartnerRecompute directory under /pod/pstore/groups/treehouse/for_core
3. Upload to Core & have Run through RNASeq/QC 3.2.1
4. Download .tar.gz and .bam output files to /pod
5. untar .tar.gz
6. move out from under UUID named folder (if time/easy)

Follow-on project: 3rd priority (after running new repository data): Re-run all repository samples on Core RNASeq/QC 3.2.1.

**Comment by klearned** [comment]
[Manifest](https://docs.google.com/spreadsheets/d/1HK_Kkej2vH6pw3Z_QvTQl0QfxbEpU4qPK9_v7vCZEwU/edit?usp=sharing) details:
Project: Expression Analysis P
Submitter Donor Primary Site ID: BTO:0000042
Submitter Specimen Type: Primary tumour - other

Count Breakdown:
Donors	132
Samples	148
Files	362
Est Upload Time (hrs)	?
Est Run Time (hrs)	51.8

**Comment by klearned** [comment]
@e-t-k I just tried creating the hard links for 362 files, but for 280 of them, I received the error: Operation Not Permitted.

I believe it is because I don't have the correct permission. Since you have magic admin powers, can you please do the necessary chmod so  the proper permissions on the files in the following list so I can create the necessary hard links? Let me know if you need anything else to do this.

/pod/pstore/groups/treehouse/for_core/exAnP/filesForChmod

**Comment by e-t-k** [comment]
@klearned Files in your list have had `chmod g+w` performed on them via docker.

At some point I'd like to train you to do this -- you have the necessary privileges and it is a straightforward task (although requiring precision).

**Comment by klearned** [comment]
Oh, you do it thru docker!!! Yes, definitely want to learn that. It would
be great to not have to bug you!

**Comment by cygao** [comment]
@e-t-k For future reference, is this the same docker chmod that you listed in #22 ?

> run id and note the numbers for your uid (eg, 30091), and a useful group id (I'd use 608, treehouse)
>Make a docker and attach the parent folder as a volume: docker run --rm -it -v /pod/parent/folder/full/path/goes/here:/data ubuntu /bin/bash
>cd /data
>You're now in the parent folder within the docker. Next, change the ownership of whatever you want. You can change it to the number of your user ID even though that user doesn't exist inside the docker. For example, this command will change ownership for everything in the parent dir and recursively the subdirs, which may or may not be what you want: chown -R 30091:608 *
>Once` you exit the docker, files will be owned by you & with the group you set & you can now chmod them without permission denial.

**Comment by klearned** [comment]
@cygao Oh cool! Thanks!!

**Comment by e-t-k** [comment]
Thanks @cygao . That's the correct guide. I gave details there on how to chown files, as that is more complicated; but you can also just chmod them as you normally would.

**Comment by klearned** [comment]
I created all the hard links and the manifest is ready. All files hard linked to this directory and manifest is there, too:
/pod/pstore/groups/treehouse/for_core/exAnP/

I sent copy of the manifest to Walt for him to look at. I also created a mini manifest to try running the first 5 samples as a test:
TH03_0285_S01
TH01_0042_S01
TH01_0043_S01
TH01_0044_S01
TH01_0045_S01

**Comment by adurbin2** [comment]
@klearned did the twelve recent samples V1 through V12 make it into this batch? If not, shall we initiate another batch for them? I'm not aware yet what the urgency is, but it's likely to be high.

**Comment by klearned** [comment]
No, they didn't. We'll do another manifest for them. I think I started one
already, but I can't remember! Let me look.

On Mar 24, 2017 1:14 PM, "adurbin2" <notifications@github.com> wrote:

@klearned <https://github.com/klearned> did the twelve recent samples V1
through V12 make it into this batch? If not, shall we initiate another
batch for them? I'm not aware yet what the urgency is, but it's likely to
be high.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/UCSC-Treehouse/operations/issues/41#issuecomment-289131716>,
or mute the thread

**Comment by klearned** [comment]
@adurbin2 OK, I remember now. I was leaving them in the manifest and going to just export the first lines for them for their own manifest. We should have a new ticket though. I need to just create the hard links and then I can submit them. I'll try to do during nap time today.

**Comment by adurbin2** [comment]
OK sounds great. But you have their TH IDs yes?

**Comment by klearned** [comment]
OK, I just deleted my last comment. I need to finalize a manifest once I get the hard links done. I have their THids that you put in the intermediate manifest.

**Comment by klearned** [comment]
OK, manifest and hard links  for the TH03 March samples are ready. Still have the big upload going. I have a question into walt to see if I can start a second upload going, so will either start upload as soon as I get an OK from him or start it after the current one is done.

**Comment by klearned** [comment]
I got the go-ahead for multiple uploads. I just started uploading the TH03 March samples.

**Comment by klearned** [comment]
First try of TH03 March samples didn't work. Waited for 360 to finish, which it did yesterday evening. Tried again today to upload the TH03 March samples, and it worked. They are all uploaded now. Expression Analysis P, the Project we assigned to all of these now shows 384 fastq.gz files uploaded, which is all of them, representing 144 donors and 160 samples. RNASeq+QC is still running. So far, 97/160 have completed RNASeq+QC.
